src/components/ViewTestDate/index.tsx

```diff
@@ -2,14 +2,14 @@ import React from 'react';
 import { Link } from '@trussworks/react-uswds';
 import { DateTime } from 'luxon';
 
-type TestDateProps = {
+type ViewTestDateProps = {
   date: DateTime;
   type: 'INITIAL' | 'REMEDIATION';
   testIndex: number;
```

Comment:
This prop name is fine. Typically when I read `index` I think of indexes starting from 0 which would be problematic here. No need to fix here, but something to think about .